src/main/java/roboy/context/dataTypes/Topic.java

```diff
@@ -0,0 +1,14 @@
+package roboy.context.dataTypes;
+
+public class Topic implements DataType {
```

Comment:
We can include timestamps this way (still to be discussed), and since Context attributes are not only String-valued, DataType is the general interface.

Reply:
timestamp and topic are 2 separate things
you are making things confusing and not intuitive